packages/client-sdk-nodejs/src/config/middleware/middleware.ts

```diff
@@ -34,6 +37,548 @@ export class MiddlewareMessage {
+            (acc, key) => acc + TEXT_DECODER.decode(key) + ', ',
+            ''
+          )
+        );
+      }
+      case cache.cache_client._IncrementRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._IncrementRequest;
+        return `IncrementRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}" and amount "${request.amount}" and ttl ${
+          request.ttl_milliseconds
+        } ms`;
+      }
+      case cache.cache_client._UpdateTtlRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._UpdateTtlRequest;
+        return `UpdateTtlRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}" and amount "${
+          String(request.increase_to_milliseconds) + ' (increase)' ||
+          String(request.decrease_to_milliseconds) + ' (decrease)' ||
+          String(request.overwrite_to_milliseconds) + ' (overwrite)'
+        }"`;
+      }
+      case cache.cache_client._ItemGetTtlRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._ItemGetTtlRequest;
+        return `ItemGetTtlRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}"`;
+      }
+      case cache.cache_client._ItemGetTypeRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._ItemGetTypeRequest;
+        return `ItemGetTypeRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}"`;
+      }
+      case cache.cache_client._DictionaryGetRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._DictionaryGetRequest;
+        return (
+          `DictionaryGetRequest with dictionary name "${TEXT_DECODER.decode(
+            request.dictionary_name
+          )}" and fields ` +
+          request.fields.reduce(
+            (acc, field) => acc + TEXT_DECODER.decode(field) + ', ',
+            ''
+          )
+        );
+      }
+      case cache.cache_client._DictionaryFetchRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._DictionaryFetchRequest;
+        return `DictionaryFetchRequest with dictionary name "${TEXT_DECODER.decode(
+          request.dictionary_name
+        )}"`;
+      }
+      case cache.cache_client._DictionarySetRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._DictionarySetRequest;
+        return (
+          `DictionarySetRequest with dictionary name "${TEXT_DECODER.decode(
+            request.dictionary_name
+          )}" and ttl ${request.ttl_milliseconds} ms and refresh ttl ${
+            request.refresh_ttl ? 'true' : 'false'
+          } and fields ` +
+          request.items.reduce(
+            (acc, item) =>
+              acc +
+              `(field "${TEXT_DECODER.decode(
+                item.field
+              )}" | value "${TEXT_DECODER.decode(item.value)}"), `,
```

Comment:
Same comment as above re: potentially truncating these values, as they might be very large and there may be a lot of them.